### ISSUE TITLE
Verification error on macOS Catalina

### ISSUE THREAD

**Issue by Chap92** [devongovett/font-manager#48]
When you try to run font-manager on macOS Catalina, you get an error: "fontmanager.node" cannot be opened because the developer cannot be verified. Will notorization resolve this issue?

**Comment by devongovett** [comment]
What do you mean by "run"? Could you give a code example? It works for me on Catalina.

**Comment by Chap92** [comment]
Does it work without getting it notarized? I tried to integrate font-manager into an Adobe CEP extension, which uses Node.js v7.7.4. font-manager was locally installed in the folder of the extension. The extension works as expected in macOS High Sierra and Mojave, but gives the above error in Catalina:  ("fontmanager.node" cannot be opened because the developer cannot be verified). I'm guessing the C++ source files have to be re-compiled and notarized for it to work correctly?

**Comment by devongovett** [comment]
Yeah I didn't do anything to notarize it. You're getting that with just `require('font-manager')`?

**Comment by Chap92** [comment]
No, the error does not come with just `require ('font-manager')`. The error pops-up when you try to invoke function `getAvailableFonts ()`:

```
	app.get ("/get-fonts", (req,res,next) => {
		fontManager.getAvailableFonts(function(fonts) { 
			res.status(200).send(fonts);
		});
	
	});
```

I'm guessing that is the point where it tries to access `fontmanager.node`. Did you make any changes to your machine to disable Gatekeeper? Like `sudo spctl --master-disable` maybe?
